lib/trento/activity_log.ex

```diff
@@ -44,14 +44,24 @@ defmodule Trento.ActivityLog do
 
   @spec list_activity_log() :: list(ActivityLog.t())
   def list_activity_log do
-    # This will be made filterable/paginatable in a later PR
     query =
       from activity in ActivityLog,
         order_by: [desc: activity.inserted_at]
 
     Repo.all(query)
   end
 
+  @spec list_activity_log(map()) :: {:ok, list(ActivityLog.t()), Flop.Meta.t()}
+  def list_activity_log(params) do
+    case Flop.validate_and_run(ActivityLog, params, for: ActivityLog) do
+      {:ok, {activity_log_entries, meta}} ->
+        {:ok, activity_log_entries, meta}
+
+      error ->
+        log_error(error, "Error while paginating activity log")
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Aren't we hiding the error here?